Makefile

```
@@ -101,11 +101,17 @@ start-postgres15:
 dev-postgres15: start-postgres15
 	cp $(CONFIG_PATH)/postgres15 $(CONFIG_FILE)
 
+start-cockroach_22_2:
+	docker compose -f docker-compose.yml up -d --remove-orphans cockroach_22_2
+
+dev-cockroach_22_2: start-cockroach_22_2
+	cp $(CONFIG_PATH)/cockroach22.2 $(CONFIG_FILE)
```

For consistency in naming:
```suggestion
	cp $(CONFIG_PATH)/cockroach_22_2 $(CONFIG_FILE)
```